### PR TITLE
Move canary related scripts to `canary/scripts`

### PR DESCRIPTION
**Pull request by wlee221**
#### Description of changes

This moves all canary related scripts to `canary/scripts`. This will ensure all the canary logics are running independent from the monorepo.

Now we can test `canary/` apps against `@next` with:

```
cd canary && yarn setup:next && yarn install && yarn build
```

#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by jacoblogan — APPROVED** [review]
LGTM can we add a note in one of our runbooks that describes how to run canaries against @next

**Review by ErikCH — APPROVED** [review]
👍

**Review by reesscot — APPROVED** [review]
LGTM. Just one question: Yarn.lock doesn't change because the dependency is still in the workspace? I thought the canaries weren't included in workspaces.

**Comment by ErikCH** [comment]
This will be updated in the  release run book?

File: canary/package.json [hunk 1]
```diff
@@ -7,6 +7,9 @@
     "build": "yarn react && yarn vue && yarn angular",
     "pull": "chmod +x ./environments/pull-environments.sh && ./environments/pull-environments.sh",
     "install": "yarn react:install && yarn vue:install && yarn angular:install",
+    "script:install": "yarn --cwd script",
+    "updatePackage": "node script/changeCanaryPackageVersion.mjs",
+    "setup:next": "rm ../postcss.config.js && yarn script:install && yarn updatePackage",
```
Comment (slaymance) [inline canary/package.json:12]: Do we also want to be able to run these commands from the root directory? Or are we ok with only running them from the `canary` dir?

**Comment by wlee221** [comment]
> Just one question: Yarn.lock doesn't change because the dependency is still in the workspace? I thought the canaries weren't included in workspaces.

Oops, tbh I forgot to update the monorepo `yarn.lock`. Now updated.

File: canary/package.json (hunk 1 above)
Comment (wlee221) [reply canary/package.json:12]: I'm okay with only running them from `canary/`, because this is an entirely different workflow from other directories.

**Pull request merged**